client/router_update.go

```diff
@@ -320,6 +320,11 @@ func (cli *VanClient) RouterUpdateVersionInNamespace(ctx context.Context, hup bo
 			// routes: skupper-controller -> skupper
 			original, err := cli.RouteClient.Routes(namespace).Get("skupper-controller", metav1.GetOptions{})
 			if err == nil {
+				siteConfig, err := cli.SiteConfigInspect(ctx, nil)
+				host := siteConfig.Spec.GetRouterIngressHost()
```

Comment:
Do we want to change this on update? I think probably not. It will not affect the certificates unless those are recreated anyway. I think the simpler and perhaps safer thing is just to say that this enhancement requires re-initialisation of the site.